go.mod

```diff
@@ -164,7 +164,7 @@ replace github.com/ethereum/go-ethereum => github.com/crypto-org-chain/go-ethere
 // TODO: remove when ibc-go and ethermint upgrades cosmos-sdk
 replace github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.44.7-0.20220214161517-8a26cd10b4be
 
-replace github.com/tharsis/ethermint => github.com/crypto-org-chain/ethermint v0.7.2-cronos-12
+replace github.com/tharsis/ethermint => github.com/crypto-org-chain/ethermint v0.7.3-0.20220307043912-20c00287a72b
```

Comment:
https://github.com/crypto-org-chain/ethermint/compare/v0.7.2-cronos-12...20c00287a72b?expand=1

Reply:
hmm looks fine but wonder why there is gomod changes

Reply:
yeah, I removed those go.mod changes, also fixes the unit test.